.github/container/Dockerfile.pax.amd64

```diff
@@ -29,6 +29,7 @@ for src in ${SRC_PATH_PAXML} ${SRC_PATH_PRAXIS}; do
   pushd ${src}
   sed -i "s| @ git+https://github.com/google/flax||g" requirements.in
   sed -i "s| @ git+https://github.com/google/jax||g" requirements.in
+  sed -i "s/etils/etils==1.7.0/g" requirements.in
```

Comment:
Could you leave a short comment on why this is needed? Future folks might be scared to remove it without knowing what it was helping